src/arviz_plots/plots/ppcplot.py

```diff
@@ -0,0 +1,529 @@
+"""ppc plot code."""
+
+from copy import copy
+from importlib import import_module
+from numbers import Integral
+
+import arviz_stats  # pylint: disable=unused-import
+import numpy as np
+from arviz_base import rcParams
+from arviz_base.labels import BaseLabeller
+from datatree import DataTree
+
+from arviz_plots.plot_collection import PlotCollection
+from arviz_plots.plots.distplot import plot_dist
+from arviz_plots.plots.utils import filter_aes, process_group_variables_coords
+from arviz_plots.visuals import labelled_title, trace_rug
+
+
+# define function
+def plot_ppc(
+    dt,
+    var_names=None,
+    filter_vars=None,
+    group="posterior",
+    observed=None,
+    observed_rug=False,
+    coords=None,
+    sample_dims=None,
+    kind=None,
+    facet_dims=None,
+    data_pairs=None,
+    aggregate=False,
+    num_pp_samples=None,
+    random_seed=None,
+    plot_collection=None,
+    backend=None,
+    labeller=None,
+    aes_map=None,
+    plot_kwargs=None,
+    stats_kwargs=None,
+    pc_kwargs=None,
+):
+    """Plot prior/posterior predictive and observed values as kde, cumulative or scatter plots.
+
+    Parameters
+    ----------
+    dt : DataTree
+        Input data with observed_data and prior/posterior predictive data.
+    var_names : str or list of str, optional
+        One or more variables to be plotted.
+        Prefix the variables by ~ when you want to exclude them from the plot.
+    filter_vars : {None, “like”, “regex”}, default=None
+        If None, interpret var_names as the real variables names.
+        If “like”, interpret var_names as substrings of the real variables names.
+        If “regex”, interpret var_names as regular expressions on the real variables names.
+    group : str, default "posterior"
+        Group to be plotted. Note: Posterior refers to posterior-predictive, prior refers to
+        prior-predictive.
+    observed : boolean, optional
+        Whether or not to plot the observed data. Defaults to True for ``group = posterior``
+        and False for ``group = prior``.
+    observed_rug : boolean, default False
+        Whether or not to plot a rug plot of the observed data. Only valid if observed=True.
+    coords : dict, optional
+        Dictionary mapping dimensions to selected coordinates to be plotted.
+        Dimensions without a mapping specified will include all coordinates for that dimension.
+        Defaults to including all coordinates for all dimensions if None.
+    sample_dims : str or sequence of hashable, optional
+        Dimensions to loop over in plotting posterior/prior predictive.
+        Note: Dims not in sample_dims or facet_dims (below) will be reduced by default.
+        Defaults to ``rcParams["data.sample_dims"]``
+    kind : {"kde", "cumulative", "scatter"}, optional
```

Comment:
```suggestion
    kind : {"kde", "cumulative", "scatter"}, optional
```

I see you did some changes like renaming "cumulative" to "ecdf" internally, but that will be confusing for users. There should be 4 possible values accepted by `kind`: kde, ecdf, hist and scatter